prime-router/src/main/kotlin/history/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -173,42 +173,17 @@ class DatabaseSubmissionsAccess(private val db: DatabaseAccess = WorkflowEngine.
         return dateFilter.and(failedFilter)
     }
 
-    fun <P, U> detailedSubmissionSelect(
-        reportsKlass: Class<P>,
-        logsKlass: Class<U>,
-    ): List<SelectFieldOrAsterisk> {
-        return listOf(
-            ACTION.asterisk(),
-            DSL.multiset(
-                DSL.select()
-                    .from(ACTION_LOG)
-                    .where(ACTION_LOG.ACTION_ID.eq(ACTION.ACTION_ID))
-            ).`as`("logs").convertFrom { r ->
-                r?.into(logsKlass)
-            },
-            DSL.multiset(
-                DSL.select()
-                    .from(REPORT_FILE)
-                    .where(REPORT_FILE.ACTION_ID.eq(ACTION.ACTION_ID))
-            ).`as`("reports").convertFrom { r ->
-                r?.into(reportsKlass)
-            },
-        )
-    }
-
     /**
      * fetch the details of a single action
      */
-    override fun <T, P, U> fetchAction(
+    override fun <T> fetchAction(
```

Comment:
Add kdoc describing what this is doing. Understandable this is not code you originally wrote, but please document it in this PRc

Reply:
done!